flink-table/flink-table-storage/src/test/java/org/apache/flink/table/storage/filestore/lsm/SortBufferMemTableWithDeduplicateAccumulatorTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.storage.filestore.lsm;
+
+import org.apache.flink.table.storage.filestore.lsm.compact.Accumulator;
+import org.apache.flink.table.storage.filestore.lsm.compact.AccumulatorTestUtils;
+import org.apache.flink.table.storage.filestore.lsm.compact.DeduplicateAccumulator;
+import org.apache.flink.table.storage.filestore.utils.ReusingTestData;
+
+import java.util.List;
+
+/** Test for {@link SortBufferMemTable} with {@link DeduplicateAccumulator}. */
+public class SortBufferMemTableWithDeduplicateAccumulatorTest extends SortBufferMemTableTestBase {
```

Comment:
Can we put these implement classes into `SortBufferMemTableTestBase`. Then we can shorten class name to `WithDeduplicateAccumulatorTest`.